scripts/compare-versions.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
When wholesale lifting code from StackOverflow, it's probably ideal to give credit. ;)

```suggestion
#!/bin/bash
# based on https://stackoverflow.com/a/4025065/65589
```